common/djangoapps/entitlements/migrations/0003_auto_20171120_1432.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Note: This needs to be removed before merging.  It could clash with another migration.  After the Policy manager PR is merged I will rebase on master and create a new migration.

Reply:
Not removed, but recreated.